### DISCUSSION TITLE
Roles

### DISCUSSION THREAD

**Discussion by djbrown** [GnuCash-Pocket/gnucash-android#19] [Team Posts]
Which Roles/Skills do we need? And how/who can carry them out?

* Project Management, Issue Tracking, Release Plan
* Android Development, Testing
* Quality Assurance, Reviews, UI-Testing, Compatibility-Testing
* Build, Pipelines, Release
* Documentation
* Community Management
* ...?

**Comment by gutierri** [comment]
I believe these and perhaps more. But we can gather what we know and apply. I believe that being a member or not, the development flow can be the same for everyone (we can even keep the contribution guide), open an issue for problems or doubts and Pull Requests for changes.

I personally can manage the project, from launch to organization and issue tracking. And also with code and pipelines as far as possible. All this through discussions here and possible proposals and suggestions in the issues (so we keep the decisions public and we can debate there ourselves). And with everyone's help.

I also believe that we can base the project management like other free/open projects that already live on this platform (GitHub) like (stow, syncthing, gnucash, remacs). Community-oriented projects.

**Comment by tpwunder** [comment]
Good morning (or whatever), 
I stumbled upon this project after realizing that GnuCash for Android is no longer in the Google Play store after upgrading my phone and discovering the app, which I use daily, was not migrated. I managed to get the app copied from my old phone to the new (even figured out how to get my data imported), but I was looking for something that could replace the now unmaintained GnuCash for Android app. 
That process led me here. But it's hard to tell if this project is actually active. I've seen some activity as recently as 4 months ago, but it certainly doesn't seem all that active.
So, is there anybody out there?

**Reply by axiopisty** [reply]
I have the same concern...

**Reply by djbrown** [reply]
for the sake of completeness https://github.com/GnuCash-Pocket/gnucash-android/issues/8#issuecomment-1998339866 😉

**Comment by axiopisty** [comment]
Hello everyone. I'm curious who here would be interested in broadening the scope of this project. I've recently learned about Flutter and am currently in a role where I have been learning and using Flutter professionally. I'm just curious if there is interest in this community for a possible rewrite of this mobile application in Flutter that would support both Android and iOS platforms.

I'm interested in taking on the challenge to do something like this. But I'm curious what kind of support there would be from the community. Specifically, I'm curious if there is anyone who would be capable and willing to collaborate by creating a Figma design for each of the screens (preferably using Material Design)?

**Reply by djbrown** [reply]
for the sake of completeness https://github.com/GnuCash-Pocket/gnucash-android/issues/8#issuecomment-1998574660 😉

**Comment by pnemonic78** [comment]
We should contact someboday relevant at the "main" GnuCash.org project about official support for the Android version.
I found an [email](gnucash-devel@gnucash.org) and [wiki](https://wiki.gnucash.org/wiki/Project_Administration)

**Reply by djbrown** [reply]
@pnemonic78 did you see https://github.com/GnuCash-Pocket/gnucash-android/issues/8#issuecomment-1998574660 already?
I was already in contact with the core team (edit: back in 2022), and got approval for the GnuCash Pocket name. But they didn't want to officialle endorse or even incorporate the project back then, but instead wanted to see if the initiative would substantiate itself first. The progress so far could be enough to affiliate to the main project. After all, they had visions of a shared core codebase between main/desktop and pocket/companion. And maybe they could help with an official google playstore account.

**Reply by djbrown** [reply]
see also https://github.com/codinguser/gnucash-android/issues/913#issuecomment-1263624322
